INSTALL.md

```diff
@@ -74,6 +78,23 @@ EOF
 
 This gatewayclass name is a parameter of the helm chart installing korifi. The helm chart is going to define a gateway that will be used for all korifi ingress traffic.
 
+#### Dynamic Provisioning
+
+Follow the dynamic provisioning [instructions](https://projectcontour.io/docs/1.26/config/gateway-api/#dynamic-provisioning) from the Gateway API support guide to install the latest version. 
+
+  - Note that as part of the Contour installation you have to create a gatewayclass with name `$GATEWAY_CLASS_NAME`:
+    ```bash
+    kubectl apply -f - <<EOF
+    kind: GatewayClass
+    apiVersion: gateway.networking.k8s.io/v1beta1
+    metadata:
+      name: $GATEWAY_CLASS_NAME
+    spec:
+      controllerName: projectcontour.io/gateway-controller
+    EOF
+    ```
+  - You DO NOT need to create a gateway class as per the instructions. The Korifi helm chart defines a gateway that will be used for all korifi ingress traffic. The gateway will be created in the `korifi-gateway` namespace.
```

Comment:
I find those two bullets a bit contradicting - do users have to create the gateway class, or not? Maybe the section should just say what users have to do.

Reply:
After re-reading the documentation, I believe that in the second bullet you meant that users do not have to create the **gateway** (instead of the ~~gateway class~~), correct?

Reply:
@danail-branekov You are correct. Sorry about that.

Reply:
Corrected.